### PR TITLE
Feature home page

### PR DESCRIPTION
**Pull request by Umuhan**
### Changes made to the code:
- The selected API  shows the list of items on the screen.
- Used API to get the total nuumber of meals.
- Set up the repository on GitHub and use Gitflow.
- Set up webpack.

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -1,39 +1,45 @@
...
+* {
   margin: 0;
   padding: 0;
+  box-sizing: border-box;
 }
 
+body {
+  /* background: url(./assets/bg.jpg);
+  background-repeat: no-repeat;
+  background-size: 100% 100%; */
```
Comment (BHObserver) [inline src/style.css]: Please remove the commented code from the codebase for best practices. Commented code makes the codebase difficult to navigate and incomprehensible.

Comment (BHObserver) [inline src/style.css]: Please remove the commented code from the codebase for best practices. Commented code makes the codebase difficult to navigate and incomprehensible.

File: src/assets/pop-bg.jpg
Comment (BHObserver) [inline src/assets/pop-bg.jpg:1]: Please consider removing the unused images.

**Review by BHObserver — CHANGES_REQUESTED** [review]
Great job following the requirements. ✨✨There are still some issues you might want to address.😊
Please check the comments below. 👇👇

**Review by BHObserver — APPROVED** [review]
Good job implementing the changes. ✨✨
It is time to merge it. ✨
Congratulations!

File: src/style.css (hunk 1 above)
Comment (Umuhan) [reply src/style.css]: done thanks.

File: src/assets/pop-bg.jpg
Comment (Umuhan) [reply src/assets/pop-bg.jpg:1]: done.

**Pull request merged**